app/models/admin_configuration.rb

```diff
@@ -37,12 +37,48 @@ class AdminConfiguration
                       unless: proc {|attributes| attributes.config_type == 'QA Dev Email'} # allow '@' for this config
 
   validate :manage_readonly_access
+  validate :ensure_docker_image_present, if: proc {|attributes| attributes.config_type == INGEST_DOCKER_NAME}
+  before_validation :strip_whitespace, if: proc {|attributes| attributes.value_type == 'String'}
 
   # really only used for IDs in the table...
   def url_safe_name
     self.config_type.downcase.gsub(/[^a-zA-Z0-9]+/, '-').chomp('-')
   end
 
+  # Ingest docker image configuration getter
+  def self.get_ingest_docker_image_config
+    AdminConfiguration.find_by(config_type: AdminConfiguration::INGEST_DOCKER_NAME)
+  end
+
+  # Ingest docker image getter
+  def self.get_ingest_docker_image
+    ingest_config = self.get_ingest_docker_image_config
+    if Rails.env != 'production' && ingest_config.present?
+      ingest_config.value
+    else
+      Rails.application.config.ingest_docker_image
+    end
+  end
+
+  # retrieve attributes from ingest docker image string
+  # example return: {registry: 'gcr.io', project: 'broad-singlecellportal-staging', image_name: 'scp-ingest-pipeline', tag: '1.5.7'}
```

Comment:
Helpful!

Reply:
This will be most useful when we add the API version check for `manage_study.py`.